### PR TITLE
docs: explain why no pdb is useful

### PR DESCRIPTION
**Pull request by prince-chrismc**
This was Diego answer to me which made a lot of sense

Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -142,6 +142,11 @@ However, there are ways to get around this, one of them is through the [/Z7](htt
 
 Adding one more common option, it seems the most simple and obvious solution, but it contains a side effect already seen with fPIC. It is necessary to manage the entire recipe, it has become a Boilerplate. So, adding PDB would be one more point to be reviewed for each recipe. In addition, in the future new options could arise, such as sanity or benchmark, further inflating the recipes. For this reason, a new option will not be added. However, the inclusion of the PDB files is discussed in issue [#1982](https://github.com/conan-io/conan-center-index/issues/1982) and there are some ideas for making this possible through a new feature. If you want to comment on the subject, please visit issue.
 
+### Doesn't this make debug build useless?
+
+No. The PDBs are only needed to debug dependency code. By providing the libraries you are able to link and build you application and debug your own code.
```
Comment (jwillikers) [inline docs/faqs.md:147]: Minor typo.

```suggestion
No. The PDBs are only needed to debug dependency code. By providing the libraries you are able to link and build your application and debug your own code.
```

File: docs/faqs.md [hunk 2]
```diff
@@ -142,6 +142,11 @@ However, there are ways to get around this, one of them is through the [/Z7](htt
 
 Adding one more common option, it seems the most simple and obvious solution, but it contains a side effect already seen with fPIC. It is necessary to manage the entire recipe, it has become a Boilerplate. So, adding PDB would be one more point to be reviewed for each recipe. In addition, in the future new options could arise, such as sanity or benchmark, further inflating the recipes. For this reason, a new option will not be added. However, the inclusion of the PDB files is discussed in issue [#1982](https://github.com/conan-io/conan-center-index/issues/1982) and there are some ideas for making this possible through a new feature. If you want to comment on the subject, please visit issue.
 
+### Doesn't this make debug build useless?
```
Comment (jwillikers) [inline docs/faqs.md:145]: Slightly correct the grammar here.

```suggestion
### Doesn't this make debug builds useless?
```

File: docs/faqs.md [hunk 3]
```diff
@@ -402,4 +407,4 @@ merging to the master branch. Feel free to contribute to a new Github Action tha
 ## Is it possible to disable Pylint?
 
 No. The [pylint](v2_linter.md) has an important role of keeping any recipe prepared for [Conan v2 migration](v2_migration.md). In case you are having
-difficult to understand [linter errors](linters.md), please, comment on your pull request about, then the community will help you.
+difficult to understand [linter errors](v2_linter.md), please, comment on your pull request about, then the community will help you.
```
Comment (jwillikers) [inline docs/faqs.md:410]:
```suggestion
difficult to understand [linter errors](v2_linter.md), please comment on your pull request about the problem to receive help from the community.
```

**Review by jwillikers — CHANGES_REQUESTED** [review]

**Review by jwillikers — APPROVED** [review]

**Review by jwillikers — APPROVED** [review]

**Review by toge — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Pull request merged**